File: src/targets/node/request/fixtures/headers.js
const request = require('request');

const options = {
<<<<<<< HEAD
<<<<<<<< HEAD:src/targets/node/request/fixtures/headers.js
  method: 'GET',
  url: 'https://httpbin.org/headers',
  headers: {accept: 'application/json', 'x-foo': 'Bar', 'x-bar': 'Foo'}
========
  method: 'POST',
  url: 'http://mockbin.com/har',
  headers: {'Content-Type': 'multipart/form-data'}
>>>>>>>> upstream/master:src/targets/node/request/fixtures/multipart-form-data-no-params.js
=======
  method: 'GET',
  url: 'http://mockbin.com/har',
  headers: {
    accept: 'application/json',
    'x-foo': 'Bar',
    'quoted-value': '"quoted" \'string\''
  }
>>>>>>> upstream/master
};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: src/targets/ruby/faraday/fixtures/headers.rb
require 'faraday'

conn = Faraday.new(
  url: 'https://httpbin.org',
)

response = conn.get('/headers') do |req|
  req.headers['accept'] = 'application/json'
  req.headers['x-foo'] = 'Bar'
  req.headers['x-bar'] = 'Foo'
  req.headers['quoted-value'] = '"quoted" \'string\''
end

puts response.status
puts response.body